web/frontend/src/lib/components/robot-client.svelte

```diff
@@ -75,7 +76,13 @@ $robotClient = new Client(impliedURL, {
   noReconnect: true,
 });
 
-let password = '';
+const passwordByAuthType: {[key in string]: string} = {}
```

Comment:
[nit] The `in` keyword is more typically used for iterating over the union of a type - I would directly use the string type.

```suggestion
const passwordByAuthType: {[key: string]: string} = {}
```